packages/sdk/src/react/ui/modals/BuyModal/_store.ts

```diff
@@ -23,6 +26,13 @@ export interface BuyModalState {
 
 export const initialState: BuyModalState = {
 	isOpen: false,
+	isCheckoutLoading: false,
+	enableIsLoadingCheckout: () => {
+		buyModal$.isCheckoutLoading.set(true);
```

Comment:
Can't we have just `isCheckoutLoading` to handle modal loading? I don't know if other two are needed. Wdyt?